activities.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+permalink: /activities
+---
+
+<style>h1, .main-wrapper h2, h3 {text-align: left; font-weight: bold;}</style>
+<h2>Goals</h2>
+* **Support** Berlin Tech Workers
+* **Build** worker power in local and international companies
+* **Contribute** to the global effort to reshape IT industry's culture and structure
+
+<h2>Activities and Services</h2>
+* **Orientation**: you are new to Germany and don't know what your rights are? You don't know who to contact to receive support during negotiations or layoffs? You want to know how to join a union or organize a worker council? Contact us.
+* **Mobilization Support**: if you want to organize the workers in your company for a strike or other forms of protest to achieve a goal specific to your company, we are here to help.
+* **Networking**: we are first and foremost a community: get in touch with like-minded tech workers that share your experiences and desires and maybe find a new job opportunity through our network.
+* **Learning**: we organize a regular learning group because we believe in empowerment through education. We are also happy to share with you our resources and knowledge in case you want to further your studies indipendently
+* **Techn(olog)ical support**: your activist group lacks the technical means to keep up with a world that is more and more digital? Do you want to integrate a critical perspective on technology in your work? TWC Berlin is open to collaboration and support with other groups and willing to share our knowledge and expertise.
+
+Our event formats:
```

Comment:
I agree with this, would be nice to include an example of each, with a link to an event.

Reply:
I added the learning group link directly to the first with a call to action. For the second I don't think we have much to show, I think.